src/VersionInfo.php

```diff
@@ -103,4 +104,8 @@ public static function VERSION() : VersionString{
 		}
 		return self::$fullVersion;
 	}
+
+	public static function BUILD_DATE() : int {
+		return time();
```

Comment:
It's supposed to be the time the phar was built, not the time the function was called.
This isn't as simple as just adding an extra function.